mgmtsystem_security_event/views/security_event_scenario.xml

```diff
@@ -0,0 +1,53 @@
+<openerp>
```

Comment:
Please use the full model name for this file: `mgmtsystem_security_event_scenario.xml`